document-store/src/integrationTest/java/org/hypertrace/core/documentstore/postgres/PostgresDocStoreTest.java

```diff
@@ -0,0 +1,159 @@
+package org.hypertrace.core.documentstore.postgres;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.hypertrace.core.documentstore.Collection;
+import org.hypertrace.core.documentstore.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.*;
+
+public class PostgresDocStoreTest {
+  public static final String ID_KEY = "id";
+  public static final String DOCUMENT_KEY = "document";
+  private static final String COLLECTION_NAME = "mytest2";
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+  private static final String UPDATED_AT = "updated_at";
+  private static final String CREATED_AT = "created_at";
```

Comment:
It seems there are assumptions on this name too - https://github.com/hypertrace/entity-service/blob/3d1f9afc4def73d9489156af16e18920e2eb17a7/entity-service-impl/src/main/java/org/hypertrace/entity/service/constants/EntityServiceConstants.java 

To me, it looks fine. we can see if there is any issue while doing e2e test. if, so, we will change its value.